### DISCUSSION TITLE
which Hoermann door does it work with ?

### DISCUSSION THREAD

**Discussion by lebeck69** [stephan192/hoermann_door#25] [Q&A]
I see 'Supramatic E3' in the text
I have a supramatic E2, any chance that this will work ?

on the net :
SUPRAMATIC E/P Série 4 > HORMANN UAP 1-**HCP**  (new generation)
If I understand correctly the HCP is for serie 4. when not specifyed, it's valid for all older version ?

I see that a official UAP1cost around 50€.
Can we connect a ESP to those and achieve the same result ?
(https://www.tor7.de/hoermann-universaladapterplatine-uap-1?number=436295)

**Comment by Gamogamer** [comment]
It does not work with the E2. I tried. If you have the know how i am sure you could make it work but I don't.

Maybe tasmota could be set up to interface with the UAP1.  If you want to program yourself there is always a way

**Comment by darmach** [comment]
It won't work with ProMatic 3 I'm afraid?

**Comment by stephan192** [comment]
Hi @darmach , have a look here https://github.com/stephan192/hoermann_door/issues/27
